src/Features/Tas/TasPlayer.cpp

```diff
@@ -334,6 +334,8 @@ void TasPlayer::Stop(bool interrupted) {
 	currentTick = 0;
 	tasControllers[0]->Disable();
 	tasControllers[1]->Disable();
+	tasFileName[0] = "";
+	tasFileName[1] = "";
```

Comment:
The issue is caused by this change. I assumed there's no reason to not reset these values after script playback, but apparently Stop() is used multiple times internally by the player itself after these values are set. Removing these solves the issue, but I might look at doing a solid refactor of how this whole playback info thing is handled.